source/include/core_pkcs11_config_defaults.h

```diff
@@ -0,0 +1,349 @@
+/*
+ * AWS IoT Device SDK for Embedded C
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+/**
+ * @file core_pkcs11_config_defaults.h
+ * @brief List of configuration macros for the corePKCS11 library along with
+ * their default values.
+ */
+
+#ifndef CORE_PKCS11_CONFIG_DEFAULTS_H_
+#define CORE_PKCS11_CONFIG_DEFAULTS_H_
+
```

Comment:
Add 
```
/* *INDENT-OFF* */
#ifdef __cplusplus
    extern "C" {
#endif
```

Reply:
As this library is included in the corePKCS11 library files, each of which already have the header guards, it is not necessarily required to add it there. We also have a mechanism present in C-SDK to catch in the future if any required file is missing the header guards for the library.

Reply:
We discussed offline, and I added the header guards in the `config_defaults.h` file for reviewer preference.